notifications/models.py

```diff
@@ -1,12 +1,23 @@
+import logging
+
 from django.db import models
 from django.urls import reverse
 from django.utils.translation import gettext_lazy as _
 from django.utils import timezone
+from django.contrib.auth import get_user_model
 
 from forum.settings import SITE_URL
 from startups.models import StartUpProfile
 from investors.models import InvestorProfile
+from users.models import Role
+from communications import init_container, MongoDBRepository
+
+
+User = get_user_model()
+logger = logging.getLogger('__name__')
 
```

Comment:
You need to either configure a logger with such a name so that it works correctly, or use an existing one:

logger = logging.getLogger('django')